### ISSUE TITLE
"Dynamic" debug launch args in the UI

### ISSUE THREAD

**Issue by GSThornton** [Microsoft/vscode#30588]
- VSCode Version: Code - Insiders 1.15.0-insider (56a5bf1724f3b882e149a3e7a24e6105a7b837c2, 2017-07-13T06:29:54.260Z)
- OS Version: Windows_NT ia32 6.1.7601
- Extensions: 

It would be great if we could specify "args" for a debug configuration from a text entry box in the editor as well as setting them inside the launch.json entry. If you are working with many launch configurations then having to keep moving args around between them as you switch is a pain. A nice text entry box undernearth the config selector in the debug pane that injected its contents into the "args" field of the config as it was launched would be awesome.

Graeme

**Comment by weinand** [comment]
You can write an extension that provides that support.
There is a discussion about this here https://github.com/Microsoft/vscode/issues/9544
Another similar request is this https://github.com/Microsoft/vscode/issues/9297

**Comment by weinand** [comment]
This feature will become available in the November release via #63987. 

Example:
```ts
{
	"version": "0.2.0",
	"configurations": [
		{
			"type": "node",
			"request": "launch",
			"name": "Launch Program",
			"program": "${workspaceRoot}/${input:programName}",
		}
	],
	"inputs": [
		{
			"id": "programName",
			"type": "pickString",
			"default": "test.js",
			"description": "The program name"
		}
	]
}
```
